lib/jazzy/sourcekitten.rb

```diff
@@ -113,7 +113,7 @@ def self.subdir_for_doc(doc, parents)
     # Builds SourceKitten arguments based on Jazzy options
     def self.arguments_from_options(options)
       arguments = ['doc']
-      unless options.module_name.empty?
+      unless options.module_name.empty? || Config.instance.objc_mode
```

Comment:
Yes, because setting the module name for an Objective-C project is still valid, it just doesn't need to be passed to SourceKitten.